ui/lib/antd.override.less

```diff
@@ -17,12 +17,3 @@
 .ant-tooltip {
   pointer-events: none;
 }
-
-// related issue:
-// Descriptions component has different look between Chrome and Safari
-// https://github.com/ant-design/ant-design/issues/24999
-// can be removed after above issue is fixed
-th.ant-descriptions-item,
-th.ant-descriptions-item-label {
```

Comment:
This change (with other ant design related change, like bumping the version) can be exacted into another PR that can be safely merged into master at any time, which will be great and can reduce the size of this PR.